ci-operator/config/openshift-eng/ocp-qe-perfscale-ci/openshift-eng-ocp-qe-perfscale-ci-main__metal-4.18-latest-x86.yaml

```diff
@@ -66,6 +67,7 @@ tests:
     - chain: openshift-qe-installer-bm-day2-label
     - chain: openshift-qe-data-path-tests
     - chain: openshift-qe-virt-density
+    - chain: openshift-qe-virt-udn-density
```

Comment:
Instead of adding the openshift-qe-virt-udn-density step into an existing job I would recommend you create a new job for testing the step.
Once it merges we can open a 2nd PR to include this step into the existing workflow.